src/Examples/UnitModels/Operations/feed_testing.ipynb

```diff
@@ -0,0 +1,376 @@
+{
+ "cells": [
+  {
+   "attachments": {
```

Comment:
@MarcusHolly I've encountered a broken HTML link with the "attachments" attribute before, and resolved the issue by using Python Image to load the file instead:

For generic image files, you can write this in Markdown and specify the image size:
"<img src='feed.png' width=\"300\" height=\"300\">"

For SVG files, it may yield better results to write this in Python code:
`from IPython.core.display import SVG
SVG(filename='feed.svg')`

Reply:
Great, thanks Brandon. I'll try this!